src/ItspServices.pServer.Persistence/ItspServices.pServer.Persistence.csproj

```diff
@@ -14,4 +14,19 @@
     <ProjectReference Include="..\ItspServices.pServer.Abstraction\ItspServices.pServer.Abstraction.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Compile Update="Properties\Resources.Designer.cs">
+      <DesignTime>True</DesignTime>
+      <AutoGen>True</AutoGen>
+      <DependentUpon>Resources.resx</DependentUpon>
+    </Compile>
+  </ItemGroup>
+
+  <ItemGroup>
+    <EmbeddedResource Update="Properties\Resources.resx">
+      <Generator>ResXFileCodeGenerator</Generator>
+      <LastGenOutput>Resources.Designer.cs</LastGenOutput>
+    </EmbeddedResource>
+  </ItemGroup>
```

Comment:
Not necessary to add a resx file here.
Simply add the DBInit.sql with:
``` XML
<ItemGroup>
  <EmbeddedResource Include="DatabaseScripts/DBInit.sql" />
</ItemGroup>
```
You can read that file then this way:
``` C#
var assembly = typeof(MyLibrary.MyClass).GetTypeInfo().Assembly;
Stream resource = assembly.GetManifestResourceStream("ItspServices.pServer.Persistence.DatabaseScripts.DBInit.sql");

```